power-management/cpufreq/cpufreq_governor/Makefile

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (c) 2019 Red Hat, Inc. All rights reserved.
+#
+# This copyrighted material is made available to anyone wishing
+# to use, modify, copy, or redistribute it subject to the terms
+# and conditions of the GNU General Public License version 2.
+#
+# This program is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+# PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public
+# License along with this program; if not, write to the Free
+# Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+# Boston, MA 02110-1301, USA.
+#
+
+export TEST		:= /kernel/power-management/cpufreq/cpufreq_governor
+export TESTVERSION	:= 1.0
+
+BUILT_FILES		= runtest
+
+FILES			= $(METADATA) \
+			  Makefile \
+			  PURPOSE \
+			  README.md \
+			  runtest.sh \
+
+.PHONY: all install download clean
+
+run: $(FILES) build
+	./runtest
+
+runtest: runtest.sh
+	cp $< $@ && chmod +x $@
+
+build: $(BUILT_FILES)
+
+clean:
+	rm -f *~ $(BUILT_FILES)
+
+include /usr/share/rhts/lib/rhts-make.include
+
+$(METADATA): Makefile
+	@echo "Owner:        Erik Hamera <ehamera@redhat.com>" > $(METADATA)
+	@echo "Name:         $(TEST)" >> $(METADATA)
+	@echo "TestVersion:  $(TESTVERSION)" >> $(METADATA)
+	@echo "Path:         $(TEST_DIR)" >> $(METADATA)
```

Comment:
Wait, hm, where is `TEST_DIR` defined?

Reply:
`TEST_DIR` is defined by file `/usr/share/rhts/lib/rhts-make.include`, 
e.g. 
```bash
$ egrep TEST_DIR= /usr/share/rhts/lib/rhts-make.include
TEST_DIR=/mnt/tests$(TEST)
```